collections/Canon_Misc/MS_Canon_Misc_575.xml

```diff
@@ -39,38 +43,118 @@
+                  </msItem>
+                  <msItem>
+                     <locus>(fol. 135v)</locus>
+                     <author key="person_61550437">Palladius of Hellenopolis</author>
+                     <title key="work_3437">Historia Lausiaca</title><bibl> (excerpt)</bibl>
+                     <incipit>Pascho quidam nomine septuagesimum agens ętatis annum in Cythi</incipit>
+                     <explicit>pacificos habui reliquos dies. Miraculum et exemplum vetus et bonum. Deo gratias.</explicit>
+                     <note>A passage of about 30 words, originally omitted, is added in the left margin: ‘latenter autem perrixi … patres veneabantur’.</note>
+                     <note>Overall, much like the version in PL, lxxiv, 346–47, and unlike the version ed. by A. Wellhausen, <title>Die lateinische Übersetzung der Historia Lausiaca des Palladius</title>, Texte und Studien, 51 (2003), pp. 573–76.</note><note>Extensively abraded and re-inked in the 14th(?) century.</note>
                   </msItem>
                </msContents>
                <physDesc>
                   <objectDesc form="codex">
                      <supportDesc material="perg">
                         <support>parchment</support>
+                        <extent> i (paper) + 1 + 136 + 1 + i (paper)
+                           <dimensions type="leaf" unit="mm">
+                              <height max="275" min="270">270-5</height>
+                              <width>200</width>
+                           </dimensions></extent>
+                        <foliation> i–iii, 1–137, in 19th-century pencil on every 5th leaf (foliation completed 2021)</foliation>
+                        <collation mainStructures="8">1(8)-17(8). The third and six leaves in quires 2, 7, 8, 9, 10 and 16 are half-sheets, as are the fourth and fifth leaves in quire 14 (i.e. fols. 107-8) and quire 15 (fols. 115-6);  the centre of quire 10 is reinforced with a later-medieval parchment strip.</collation>
                      </supportDesc>
+                     <layoutDesc>
+                        <layout columns="1" writtenLines="30" rulingMedium="hardpoint">Blind-ruled for 30 lines per page. Ruled space <dimensions type="ruled" unit="mm">
+                           <height>210</height>
+                           <width>135</width>
+                        </dimensions></layout>
+                     </layoutDesc>
                   </objectDesc>
+                  <handDesc>
+                     <handNote script="minusculeCaroline" scope="major">Caroline minuscule; incipits in majuscules.</handNote>
+                     <handNote script="capitalsRustic" scope="minor">Rustic capitals for rubrics and fol. 122.</handNote>
+                  </handDesc>
+                  <musicNotation>
+                     <p>Neumes added above an added marginal antiphon for the feast of St Gregory: ‘Evovae Sex strux(it) in sicilia vir clarus monasteria’ (fol. 122v).</p>
```

Comment:
'Euouae' is the tonality guide, not part of the antiphon. The text is Cantus 004878/CAO 4878, <http://www.cantusindex.org/id/004878>.